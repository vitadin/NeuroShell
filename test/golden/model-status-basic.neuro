%% Test basic model status functionality
\set[_echo_command="true"]

%% Create multiple models with different providers
\model-new[catalog_id="O4M"] claude-model
\model-new[catalog_id="O3"] openai-model
\model-new[catalog_id="O4M", description="Custom Claude model"] custom-claude

%% Test basic model status command
\model-status

%% Verify system variables are set correctly
\get[#model_count]
\get[#model_filtered_count]
\get[_output]