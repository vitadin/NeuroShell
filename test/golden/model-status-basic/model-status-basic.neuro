%% Test basic model status functionality
\set[_echo_command="true"]

%% Create multiple models with different providers
\model-new[catalog_id="CS4"] claude-model
\model-new[catalog_id="O3"] openai-model
\model-new[provider="anthropic", base_model="claude-3-sonnet", description="Custom Claude model"] custom-claude

%% Test basic model status command
\model-status

%% Verify system variables are set correctly
\get[#model_count]
\get[#model_filtered_count]
\get[_output]