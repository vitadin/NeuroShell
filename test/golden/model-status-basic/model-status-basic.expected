Setting _echo_command = true
%%> "\\model-new[catalog_id=\"CS4\"] claude-model"
Created model 'claude-model' (ID: 00000001, Provider: anthropic, Base: claude-sonnet-4-20250514)
%%> "\\model-new[catalog_id=\"O3\"] openai-model"
Created model 'openai-model' (ID: 00000002, Provider: openai, Base: o3)
%%> "\\model-new[provider=\"anthropic\", base_model=\"claude-3-sonnet\", description=\"Custom Claude model\"] custom-claude"
Created model 'custom-claude' (ID: 00000003, Provider: anthropic, Base: claude-3-sonnet)
%%> "\\model-status"
Model Configurations (3 models):
  claude-model (00000001)
    Provider: anthropic
    Base Model: claude-sonnet-4-20250514
    Catalog ID: CS4
    Parameters: none
    Created: 2025-01-01 00:00:01
  custom-claude (00000003)
    Provider: anthropic
    Base Model: claude-3-sonnet
    Catalog ID: none
    Parameters: none
    Created: 2025-01-01 00:00:03
    Description: Custom Claude model
    Active: true
  openai-model (00000002)
    Provider: openai
    Base Model: o3
    Catalog ID: O3
    Parameters: none
    Created: 2025-01-01 00:00:02
%%> "\\get[#model_count]"
#model_count = 3
%%> "\\get[#model_filtered_count]"
#model_filtered_count = 3
%%> "\\get[_output]"
_output = Model Configurations (3 models):
  claude-model (00000001)
    Provider: anthropic
    Base Model: claude-sonnet-4-20250514
    Catalog ID: CS4
    Parameters: none
    Created: 2025-01-01 00:00:01
  custom-claude (00000003)
    Provider: anthropic
    Base Model: claude-3-sonnet
    Catalog ID: none
    Parameters: none
    Created: 2025-01-01 00:00:03
    Description: Custom Claude model
    Active: true
  openai-model (00000002)
    Provider: openai
    Base Model: o3
    Catalog ID: O3
    Parameters: none
    Created: 2025-01-01 00:00:02